pkg/scheduler/tests/scheduler_smoke_test.go

```diff
@@ -1364,131 +1456,124 @@ partitions:
     assert.Equal(t, int(totalNode2Resource), 0)
 }
 
-//func TestScheduleBestEffortRequests(t *testing.T) {
```

Comment:
can we do not remove this for now?
I think we need some tests to cover best effort request, can't remember what was exactly the reason this was commented out, but it would be nice to work on this and get this part into test coverage.

Reply:
The test is never going to work as we cannot schedule a request with all resources set to 0. To allow that we need to properly think about it. We did a shortcut for QOS pods for K8s as per #54.

We moved the real zero resource support issue out. I opened a new issue for this #75.  When we implement that we need to add new tests that test the implementation we then add.